### PR TITLE
Log OCP and CNV versions

### PR DESCRIPTION
**Pull request by arnongilboa**
Also go mod tidy && go mod vendor (required for the OpenShift APIs)

### CODE REVIEW

File: pkg/internal/checkup/checkup.go [hunk 1]
```diff
@@ -180,6 +187,36 @@ func (c *Checkup) Run(ctx context.Context) error {
...
+			ocpVersion = update.Version
+			break
+		}
+	}
+
+	cdis, err := c.client.ListCDIs(ctx)
+	if err != nil {
+		return err
+	}
+	if len(cdis.Items) != 1 {
```
Comment (ShellyKa13) [inline pkg/internal/checkup/checkup.go:210]: maybe check if ==0 then have an error that it doesnt exist and if bigger then one then this current msg?

File: pkg/internal/checkup/checkup.go [hunk 2]
```diff
@@ -180,6 +187,36 @@ func (c *Checkup) Run(ctx context.Context) error {
...
+func (c *Checkup) checkVersions(ctx context.Context, errStr *string) error {
+	log.Print("checkVersions")
+
+	ver, err := c.client.GetClusterVersion(ctx, "version")
+	if err != nil {
+		return err
+	}
+	ocpVersion := ""
+	for _, update := range ver.Status.History {
+		if update.State == configv1.CompletedUpdate {
```
Comment (ShellyKa13) [inline pkg/internal/checkup/checkup.go:199]: and there is always only on update that has that state? or if not are they sorted by old->new? cause if not you will not get the last version necessarily. Just making sure :)

File: pkg/internal/checkup/checkup_test.go [hunk 3]
```diff
@@ -621,6 +623,37 @@ func (cs *clientStub) GetDataSource(ctx context.Context, namespace, name string)
 	return das, nil
 }
 
+func (cs *clientStub) GetClusterVersion(ctx context.Context, name string) (*configv1.ClusterVersion, error) {
```
Comment (ShellyKa13) [inline pkg/internal/checkup/checkup_test.go:626]: what about a UT to make sure you get the last updated version? and maybe also negative testing.. although it is pretty straight foreword function..

File: pkg/internal/checkup/checkup.go (hunk 2 above)
Comment (arnongilboa) [reply pkg/internal/checkup/checkup.go:199]: See doc: `The newest update is first in the list and it is ordered by recency.`

Comment (ShellyKa13) [reply pkg/internal/checkup/checkup.go:199]: missed the `break` :)

File: pkg/internal/checkup/checkup_test.go (hunk 3 above)
Comment (arnongilboa) [reply pkg/internal/checkup/checkup_test.go:626]: Will add the positive one :)

**Review by ShellyKa13 — APPROVED** [review]
Look good!

**Pull request merged**